File: 백준/Bronze/28702. FizzBuzz/FizzBuzz.py
a = input().strip()
b = input().strip()
c = input().strip()

if a.isdigit():
    num = int(a) + 3
elif b.isdigit():
    num = int(b) + 2
else:
    num = int(c) + 1

if num % 3 == 0 and num % 5 == 0:
    print("FizzBuzz")
elif num % 3 == 0:
    print("Fizz")
elif num % 5 == 0:
    print("Buzz")
else:
    print(num)

File: 리트코드/Easy/412. Fizz Buzz/Fizz Buzz.js
var fizzBuzz = function(n) {
    let result_arr = [];
    for (let i=1; i<=n; i++){
        if (i % 3 == 0 && i % 5 == 0){
            result_arr.push("FizzBuzz");
        }
        else if (i % 3 == 0){
            result_arr.push("Fizz");
        }
        else if (i % 5 == 0){
            result_arr.push("Buzz");
        }
        else{
            result_arr.push(i.toString());
        }
    }
    return result_arr;
};
